djconnectwise/sync.py

```diff
@@ -854,6 +854,12 @@ def _assign_field_data(self, instance, json_data):
             instance.inactive = json_data.get('inactive')
 
         instance.project_flag = json_data.get('projectFlag', False)
+        instance.time_entry_discussion_flag = \
+            json_data.get('timeEntryDiscussionFlag', False)
+        instance.time_entry_resolution_flag = \
+            json_data.get('timeEntryResolutionFlag', False)
+        instance.time_entry_internal_analysis_flag = \
+            json_data.get('timeEntryInternalAnalysisFlag', False)
```

Comment:
We have PEP8 checking that runs during `make test` and these lines had to be shortened cause they don't meet PEP8 line length guidelines. Make sure to run `make test` before putting your code up for review.